.github/workflows/test.yml

```diff
@@ -69,7 +69,7 @@ jobs:
         key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
         restore-keys: ${{ runner.os }}-go-
     - name: Run Docker containers
-      run: docker-compose up --detach
+      run: docker-compose --env-file .github/workflows/.env up --detach
       env:
```

Comment:
agreed. no need to modify this command. we can modify the docker-compose.yml directly if that's the file being used here to spin-up containers.

> AFAIK it wont break but OPENSEARCH_INITIAL_ADMIN_PASSWORD would just redundant for old versions right?

Correct. This variable will only be used for 2.12 and above. Else it would simply not be used.